_posts/en/newsletters/2019-04-02-newsletter.md

```diff
@@ -0,0 +1,269 @@
+---
+title: 'Bitcoin Optech Newsletter #40'
+permalink: /en/newsletters/2019/04/02/
+name: 2019-04-02-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter notes a spike in estimated transaction fees,
+describes LN trampoline payments, and publicizes Bitcoin Core's intent
+to default its built-in wallet to bech32 receiving addresses in version
+0.20 or earlier.  Also included are regular sections about bech32
+sending support and notable code changes in popular Bitcoin
+infrastructure projects.
+
+## Action items
+
+- **Help test Bitcoin Core 0.18.0 RC2:** The second Release Candidate
+  (RC) for the next major version of Bitcoin Core has been [released][0.18.0].
+  Testing is still needed by organizations and experienced users who
+  plan to run the new version of Bitcoin Core in production.  Use [this
+  issue][Bitcoin Core #15555] for reporting feedback.
+
+## Network status
+
+- **Fee increases for fast confirmation:** after over a year of most
+  Bitcoin transactions confirming rather quickly as long as they paid a
+  feerate above the default minimum relay fee (except during [a brief
+  exceptional period][Newsletter #22]), a modest backlog has developed
+  over the previous week and raised the feerates for people who need
+  their transactions to confirm within the next several blocks.
+  Spenders willing to wait a bit longer can still receive a good deal.
+  For example, as of this writing, Bitcoin Core's fee estimator suggests
+  a fee per 1,000 vbytes of 533 µBTC for confirmation within 2 blocks
+  but only 25 µBTC for confirmation within 50 blocks---a 95% savings for
+  waiting up to an extra 8 hours.  For more information, we recommend
+  [Johoe's mempool statistics][] and [P2SH.info's fee estimate
+  tracker][].
+
+## News
+
+- **Trampoline payments for LN:** Pierre-Marie Padiou started a
+  [thread][trampoline thread] on the Lightning-Dev mailing list
+  suggesting that Alice could send a payment to Zed even if she didn't
+  know a path to his node by first sending a payment to an intermediate
+  node (Dan) and asking Dan to figure out the route the rest of the way
+  to Zed.  This would especially benefit Alice if she ran a lightweight
+  LN node that didn't attempt to keep track of the entire network.  For
+  increased privacy, Alice could use several intermediate nodes in
+  succession rather than just one (each one receiving it's own
```

Comment:
I think you interpreted it the same way I read the email, and I don't understand how your suggested change affects the meaning.  I am removing "in succession" because it actually doesn't add anything to the sentence.